src/main/java/duke/command/RestoreCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.Ui;
+import duke.task.TaskList;
+
+/**
+ * The RestoreCommand is given when the user wants to revert to a previously saved TaskList.
+ */
+public class RestoreCommand extends Command {
```

Comment:
I like how you took into account the situation where the list is accidentally deleted. :)